PWGCF/MultiparticleCorrelations/Tasks/O2AnalysisTaskThreePartCorr.cxx

```diff
@@ -0,0 +1,299 @@
+            }
+          }
+        }
+      }
+    }
+    // End of the V0-Track Correlations
+  }
+  PROCESS_SWITCH(ThreePartCorr, processSame, "Process same-event correlations", true);
+
+  void processMixed(MyFilteredCollisions const& collisions, MyFilteredV0s const& v0s, MyFilteredTracks const& tracks)
+  {
+
+    // Start of the Mixed-events Correlations
+    for (const auto& [coll_1, v0_1, coll_2, track_2] : pair) {
+      for (const auto& [trigger, associate] : soa::combinations(soa::CombinationsFullIndexPolicy(v0_1, track_2))) {
+        if (V0Filters(trigger) && TrackFilters(trigger, associate)) {
+
+          TSign = V0Sign(trigger);
+          if (TSign == 1) {
+            CandMass = trigger.mLambda();
+          } else if (TSign == -1) {
+            CandMass = trigger.mAntiLambda();
+          }
+
+          DeltaPhi = PhiCorrection(trigger.phi(), associate.phi());
+          DeltaEta = trigger.eta() - associate.eta();
+
+          if (CandMass >= 1.10 && CandMass <= 1.13) {
+            if (TrackPID(associate).first == track::PID::Pion) {
+              MECorrRegistry.fill(HIST("hMixLambdaPion_SGNL"), DeltaPhi, DeltaEta, coll_1.centFT0C(), coll_1.posZ(), trigger.pt(), associate.pt(), TSign, associate.sign());
+            } else if (TrackPID(associate).first == track::PID::Kaon) {
+              MECorrRegistry.fill(HIST("hMixLambdaKaon_SGNL"), DeltaPhi, DeltaEta, coll_1.centFT0C(), coll_1.posZ(), trigger.pt(), associate.pt(), TSign, associate.sign());
+            } else if (TrackPID(associate).first == track::PID::Proton) {
+              MECorrRegistry.fill(HIST("hMixLambdaProton_SGNL"), DeltaPhi, DeltaEta, coll_1.centFT0C(), coll_1.posZ(), trigger.pt(), associate.pt(), TSign, associate.sign());
+            }
+          } else {
+            if (TrackPID(associate).first == track::PID::Pion) {
+              MECorrRegistry.fill(HIST("hMixLambdaPion_SB"), DeltaPhi, DeltaEta, coll_1.centFT0C(), coll_1.posZ(), trigger.pt(), associate.pt(), TSign, associate.sign());
+            } else if (TrackPID(associate).first == track::PID::Kaon) {
+              MECorrRegistry.fill(HIST("hMixLambdaKaon_SB"), DeltaPhi, DeltaEta, coll_1.centFT0C(), coll_1.posZ(), trigger.pt(), associate.pt(), TSign, associate.sign());
+            } else if (TrackPID(associate).first == track::PID::Proton) {
+              MECorrRegistry.fill(HIST("hMixLambdaProton_SB"), DeltaPhi, DeltaEta, coll_1.centFT0C(), coll_1.posZ(), trigger.pt(), associate.pt(), TSign, associate.sign());
+            }
+          }
+        }
+      }
+    }
+    // End of the Mixed-events Correlations
+  }
+  PROCESS_SWITCH(ThreePartCorr, processMixed, "Process mixed-event correlations", true);
+
+  //================================================================================================================================================================================================================
+
+  Double_t PhiCorrection(Double_t TriggerPhi, Double_t AssociatePhi)
```

Comment:
`Correction` is misleading because nothing is corrected
I would suggest `DeltaPhiAdjust` or `DeltaPhiShift`